packages/stream_chat_flutter/lib/src/stream_chat.dart

```diff
@@ -95,52 +95,73 @@ class StreamChatState extends State<StreamChat> {
   StreamChatConfigurationData get streamChatConfigData =>
       widget.streamChatConfigData ?? StreamChatConfigurationData();
 
+  /// Whether [VideoPlayerMediaKit.ensureInitialized] has been invoked.
+  static bool _mediaKitInitialized = false;
+
   @override
   void initState() {
     super.initState();
-    // Ensures that VLC only initializes in real desktop environments
-    if (!isTestEnvironment && isDesktopVideoPlayerSupported) {
-      VlcManager.instance.initialize();
+    // package:video_player does not support Microsoft Windows & GNU/Linux.
+    // package:video_player_media_kit bridges support for these platforms using
+    // package:media_kit.
+    if (!_mediaKitInitialized && isDesktopVideoPlayerSupported) {
+      _mediaKitInitialized = true;
```

Comment:
Shouldn't this happer after `ensureInitialized` is invoked, as mentioned in the variable's comment?

Reply:
I made the changes upstream to internally check for multiple initializations: https://github.com/media-kit/media-kit/commit/6516ef12cb7c117ad670c64901efdfa6c344af44.

This eliminates the requirement of previous additional check & static variable.